### PR TITLE
Added IPv6 support for syslog_server

### PR DESCRIPTION
**Pull request by jonnytdevops**
TestSyslogServer#test_create_destroy_single_ipv6 = 13.91 s = .
TestSyslogServer#test_create_destroy_single = 12.78 s = .
TestSyslogServer#test_create_destroy_multiple = 12.90 s = .
TestSyslogServer#test_create_destroy_single_vrf = 13.75 s = .

Finished in 53.347300s, 0.0750 runs/s, 0.6561 assertions/s.

4 runs, 35 assertions, 0 failures, 0 errors, 0 skips

### CODE REVIEW

File: tests/test_syslog_server.rb [hunk 1]
```diff
@@ -35,36 +35,48 @@ def teardown
...
     config('no logging server 1.2.3.4',
-           'no logging server 2.3.4.5',
+           'no logging server 2003::2',
            'no vrf context red')
   end
 
   # TESTS
 
-  def test_syslogserver_create_destroy_single
+  def test_create_destroy_single
```
Comment (glennmatthews) [inline tests/test_syslog_server.rb]: Might as well call this `test_create_destroy_single_ipv4` for clarity.

File: tests/test_syslog_server.rb [hunk 2]
```diff
@@ -72,7 +84,7 @@ def test_syslogserver_create_destroy_multiple
     refute_includes(Cisco::SyslogServer.syslogservers, id2)
   end
 
-  def test_syslogserver_create_destroy_single_vrf
+  def test_create_destroy_single_vrf
```
Comment (glennmatthews) [inline tests/test_syslog_server.rb]: Do we need a VRF IPv6 test to be added?

**Comment by jonnytdevops** [comment]
TestSyslogServer#test_create_destroy_multiple = 14.05 s = .
TestSyslogServer#test_create_destroy_single_vrf_ipv4 = 13.86 s = .
TestSyslogServer#test_create_destroy_single_vrf_ipv6 = 12.90 s = .
TestSyslogServer#test_create_destroy_single_ipv4 = 12.85 s = .
TestSyslogServer#test_create_destroy_single_ipv6 = 12.85 s = .

Finished in 66.502372s, 0.0752 runs/s, 0.6316 assertions/s.

5 runs, 42 assertions, 0 failures, 0 errors, 0 skips

**Comment by jonnytdevops** [comment]
@glennmatthews code updated

**Pull request merged**